features/facebookImport/src/views/overview/overview.css

```diff
@@ -29,60 +39,40 @@ h4 {
     font-size: 16px;
 }
 
-.overview p {
-    margin: 0px;
-    font-size: 14;
-}
-
-.overview .details p {
-    font-size: 14px;
+.overview .bubble-cluster {
+    display: flex;
+    overflow: hidden;
+    flex-grow: 1;
 }
 
-.overview .details .size {
-    color: var(--color-gray-400);
+.overview .bubble-cluster svg {
+    flex-grow: 1;
 }
 
-.overview .details .separator {
-    width: 100%;
-    height: 1px;
-    box-sizing: border-box;
-    background-color: var(--color-gray-400);
-    margin: 20px 0px 16px;
-}
-
-.overview .imported-files {
-    overflow-y: scroll;
-    height: 100%;
+.overview p {
+    margin: 0px;
+    font-size: 14;
```

Comment:
I suppose it works without specifying the unit (i.e. `14px`), but not sure, so I felt like pointing it out :D

Reply:
Not sure where this is coming from, but I've changed it anyways to avoid future issues (and because the font-size is actually `16px`).